galaxy_ng/tests/functional/cli/test_collection_upload.py

```diff
@@ -44,3 +45,54 @@ def test_upload_collection(self):
         self.assertEqual(collection.namespace, "pulp")
         self.assertEqual(collection.name, "squeezer")
         self.assertEqual(collection.highest_version["version"], "0.0.9")
+
+        # Cleanup
+        self.delete_collection(collection.namespace, collection.name)
+        self.delete_namespace(collection.namespace)
+
+    @skip("Requires GALAXY_ENABLE_API_ACCESS_LOG to be enabled")
```

Comment:
I think before this PR can be merged we need to either
- Find/build way to export env var GALAXY_ENABLE_API_ACCESS_LOG so this test is not skipped
- Move this test to IQE, test against pr branch, merge test